.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @dimberman @tatiana @pankajkoti
+* @astronomer/astro-devex
```

Comment:
I believe we can include DevEx without excluding the original creators/maintainers. Pankaj's reviews have allowed us to iterate much faster

Reply:
@tatiana updated!